src/box.tsx

```diff
@@ -14,6 +15,35 @@ const Box: BoxComponent = ({ is = 'div', innerRef, children, ...props }) => {
     parsedProps.ref = innerRef
   }
 
+  /**
+   * If the user has enabled safe hrefs we want to make sure that the url passed
+   * is safe and that the other attributes that make the link safe are added to the
+   * element
+   */
+  if (getUseSafeHref() && is === 'a' && parsedProps.href) {
```

Comment:
Also, do we want to expose this as a _prop_ too? Seems like you may not only want to set the default global behavior (what you already have), you may also want to opt-in on an individual basis?

That prop could provide an easy forwards-compatible pivot for turning safety on by default and letting users opt-out on a per-component basis, e.g. `allowUnsafeHref`

Reply:
(Could always add that later)

Reply:
This seems like a solid idea. I could see potential situations, where generally you want all the safety, but there may be a situation where you want to forego it in a singular situation